### ISSUE TITLE
Feature request: Add a command to open the file under test or test for file

### ISSUE THREAD

**Issue by seanpoulter** [jest-community/vscode-jest#233]
### Expected Behavior

I'd love to have a command opens the related test or file for the current document. This should work both ways:
- When editing source code, open the test
- When editing a test, open the source code
Since we won't be certain, the command could open up a quick pick menu showing the imported files.

I'm not sure if that falls under the scope of this extension, but I find myself wishing this was a thing quite often so I'm pretty motivated to add it.

**Comment by orta** [comment]
Yeah, makes sense to me to be in here. This code exists somewhere in jest, note that there can be multiple test files for one file, so ideally looping through them>

**Comment by seanpoulter** [comment]
Good point @orta. I'll see if there's something I can pull in from Jest.
